### ISSUE TITLE
Merge with core repository

### ISSUE THREAD

**Issue by borgmanJeremy** [flameshotapp/packages#13]
Do we want to keep this as a separate repo now that we have full control of the source code? I was thinking we could add it directly to the core repo with the rpm and deb packages.

**Comment by holazt** [comment]
Resolved at PR https://github.com/flameshot-org/flameshot/pull/851 and https://github.com/flameshot-org/flameshot/pull/866. I think this repo could be **archieved** or deleted. And this repo https://github.com/flameshot-org/flathub should be deleted. @mmahmoudian  @borgmanJeremy  What's your opinion?

**Comment by mmahmoudian** [comment]
I agree, Now this repo doesn't serve a purpose as we have access to the core repo. Historically it has some value as shows a creative attempt to solve something, but now having it merged with core sounds and looks better and more cohesive imho.

I checked and i don't have permission to archive/delete. @lupoDharkael and perhaps @borgmanJeremy can handle it.

**Comment by borgmanJeremy** [comment]
Yes lets archive it. I do not have permission to do this so we need @lupoDharkael to grant someone permissions. It's not an urgent request.
